docs/agents/holo_fpga.rst

```diff
@@ -0,0 +1,69 @@
+.. highlight:: rst
+
+.. _holo_fpga_agent:
+
+========================
+Holography FPGA Agent
+========================
+
+The Holography FPGA Agent is provided with OCS to help demonstrate and debug issues with the holography ROACH2 FPGA.
+It will connect the computer to the ROACH via an ethernet port, take data, and pass it to the OCS feed.
+
+.. argparse::
+   :module: agents.holo_fpga.roach_agent
+   :func: make_parser
+   :prog: roach_agent.py
+
+Dependencies
+------------
+
+Python Packages:
+- casperfpga (https://pypi.org/project/casperfpga/)
+- holo_daq (https://github.com/McMahonCosmologyGroup/holog_daq)
```

Comment:
The bulleted list needs a blank line before it to render properly.

Also, the links can be made with:
```
- `casperfpga <https://pypi.org/project/casperfpga/>`_
```

which will show as:

- [casperfpga](https://pypi.org/project/casperfpga/) 

rather than:

- casperfpga (https://pypi.org/project/casperfpga/)

(Same comment for the holo_synth page.)